templates/kube-state-metrics/030-Deployment-kube-state-metrics.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: enmasse
+  name: kube-metrics
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: enmasse
+      name: kube-metrics
```

Comment:
Should we use name or component? I would prefer component if we're moving towards that.